File: FLUTTER/igest_app/lib/models/principal/departamento.dart
class Departamento {
  String idDepartamento;
  int idPais;
  String nomDepartamento;

  Departamento({
    required this.idDepartamento,
    required this.idPais,
    required this.nomDepartamento,
  });

  factory Departamento.fromJson(Map<String, dynamic> json) {
    return Departamento(
      idDepartamento: json['id_departamento'],
      idPais: json['id_pais'],
      nomDepartamento: json['nom_departamento'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id_departamento': idDepartamento,
      'id_pais': idPais,
      'nom_departamento': nomDepartamento,
    };
  }
}

File: SPRING/igest.server.model/src/main/java/com/yupanastudio/igest/server/principal/dml/Departamento.java
/*
 * Document   : Departamento
 * Author     : Franklin Barzola Olivares <frankoantonio@gmail.com>
 */
package com.yupanastudio.igest.server.principal.dml;

import org.springframework.data.annotation.Id;

public class Departamento {

    @Id
    private String idDepartamento;
    private int idPais;
    private String nomDepartamento;

    public String getIdDepartamento() {
        return idDepartamento;
    }

    public void setIdDepartamento(String idDepartamento) {
        this.idDepartamento = idDepartamento;
    }

    public int getIdPais() {
        return idPais;
    }

    public void setIdPais(int idPais) {
        this.idPais = idPais;
    }

    public String getNomDepartamento() {
        return nomDepartamento;
    }

    public void setNomDepartamento(String nomDepartamento) {
        this.nomDepartamento = nomDepartamento;
    }

}
